test/lib/ansible_test/_internal/cloud/vcenter.py

```diff
@@ -51,10 +51,11 @@ def __init__(self, args):
         # file or hosted in worldstream.  Using an env var value of 'worldstream' with appropriate
         # CI credentials will deploy a dynamic baremetal environment. The simulator is the default
         # if no other config if provided.
-        self.vmware_test_platform = os.environ.get('VMWARE_TEST_PLATFORM', '')
+        self.vmware_test_platform = os.environ.get('VMWARE_TEST_PLATFORM', 'govcsim')
         self.aci = None
```

Comment:
This no longer accepts `VMWARE_TEST_PLATFORM=''`. The var be unset or set to worldstream or govcsim.  This is probably fine, just want to note the change.

Reply:
Yes, this is done on purpose.